lib/membrane_live_web/helpers/controller_callback_helper.ex

```diff
@@ -4,14 +4,26 @@ defmodule MembraneLiveWeb.Helpers.ControllerCallbackHelper do
 
   alias MembraneLive.Webinars
 
-  def get_with_callback(
+  def get_webinar_and_fire_callback(
+        conn,
+        params,
+        callback,
+        show_callback? \\ false,
+        products? \\ false
```

Comment:
I think name `include_products?` would be better, but if you have a strong preference for this name it can stay as it is.